metis-normalization/src/main/java/eu/europeana/normalization/normalizers/DatesNormalizer.java

```diff
@@ -240,6 +241,7 @@ public DateNormalizationResult normalizeDateProperty(String input) {
         this::noMatchIfValidAndTimeOnly);
   }
 
+
   // TODO: 21/07/2022 To check. This method does not do dates switching like the other method
```

Comment:
Yes that is correct. We don't need to do the switching, we are stricter on generic properties.